Sources/AppStoreConnectClient/AppStoreConnectClient.swift

```diff
@@ -55,45 +55,73 @@ public struct AppStoreConnectClient {
         let response = try await client.apps_hyphen_get_collection()
         switch response {
         case .ok(let okResponse):
-            switch okResponse.body {
-            case .json(let json):
-                return json.data.compactMap({ Application(schema: $0) })
-            }
+            return try okResponse.body.json.data.compactMap({ Application(schema: $0) })
         case .badRequest(let result):
-            throw AppStoreConnectError.badRequest
+            throw AppStoreConnectError.badRequest(
```

Comment:
i believe that we can reuse "errorResponse" here too

Reply:
Refactored.